src/database.ts

```diff
@@ -1,11 +1,14 @@
 import fs from 'node:fs';
-import dotenv from 'dotenv';
-import { knex } from 'knex';
-import { User, Word } from './interfaces/Database';
 import dayjs from 'dayjs';
+import utc from 'dayjs/plugin/utc';
 import timezone from 'dayjs/plugin/timezone';
-import { log } from './utils/log';
+import dotenv from 'dotenv';
+import { knex } from 'knex';
+import { User, Word } from './interfaces/database';
```

Comment:
```suggestion
import { User, Word } from './interfaces/Database';
```

This was causing some of the errors.

Reply:
This actually isn't needed anymore, since I created a `index.ts` file in this folder too. But I will update!